diff --git a/core/arc-middle/mng/action.c b/core/arc-middle/mng/action.c
index 3eb6c67..317e1be 100644
--- a/core/arc-middle/mng/action.c
+++ b/core/arc-middle/mng/action.c
@@ -24,6 +24,13 @@
 #define BUF_FULL_STR " ...\n(Buffer is full, unable to receive all data.)\n"
 #define BUF_FULL_STR_SZ sizeof(BUF_FULL_STR)
 
+#define cprintf(fmt, args...) do { \
+	FILE *fp = fopen("/dev/console", "w"); \
+	if (fp) { \
+		fprintf(fp, fmt, ## args); \
+		fclose(fp); \
+	} \
+} while (0)
 int send_msg(const unsigned char * msg, int len)
 {
     int s;
@@ -70,12 +77,17 @@ int mng_action(const char * action, const char * param)
 	request_action_p req;
 	int len;
 	int ret = -1;
+	char cmd[128] = {0};
 
 	if(action == NULL || action[0] == '\0')
 	{
 		return 0;
 	}
 
+	sprintf(cmd, "echo %s >> /tmp/actions", action);
+	system(cmd);
+	cprintf("allenlog: action:%s\n",action);
+
 	if(param == NULL)
 	{
 		param = "";
